dipy/viz/actor.py

```diff
@@ -201,6 +201,139 @@ def copy(self):
     return image_actor
 
 
+def contour_actor(data, affine=None, levels=[50],
+            colors=[np.array([1.0, 0.0, 0.0])], opacities=[1]):
```

Comment:
Thanks, ariel! I just added PEP8 comments to my pycharm so this should happen less. It doesn't like the default values I set; is this fine for dipy or is there another way I should do this? (It hilights levels, colors, and opacities with this comment: "This inspection detects when a mutable value as list or dictionary is detected in a default value for an argument. Default argument values are evaluated only once at function definition time, which means that modifying the default value of the argument will affect all subsequent calls of the function. ") Thanks! -Kesshi